src/Esprima/Ast/BigIntLiteral.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Numerics;
+
+namespace Esprima.Ast
+{
+    public sealed class BigIntLiteral : Literal
+    {
+        public readonly string BigInt;
+
+        public BigInteger? BigIntValue => (BigInteger?) Value;
```

Comment:
do we really need to add this as Literal already has that, I think it should be quite cheap to call even with the type check

Reply:
I don't need it, I only did to match the other...

Reply:
I did it to match the Tree spec: https://github.com/estree/estree/blob/master/es2020.md#bigintliteral

Reply:
Caus I think the AST should be nearly the same in all Javascript Parsers